src/containers/pages/FocusInvestigation/map/planCompletion/tests/__snapshots__/index.test.tsx.snap

```diff
@@ -25,14 +25,7 @@ exports[`@containers/pages/map/planCompletion/ renders correctly 1`] = `
   <h2
     className="mb-3 mt-5 page-title"
   >
-    Mark 
-    <span
-      className="plan-name"
-    >
-      $
-      A1-Tha Luang Village 1 Focus 01
-    </span>
-     as complete
+    Mark A1-Tha Luang Village 1 Focus 01 as complete
```

Comment:
This implementation does not support display strings with composite HTML.

Reply:
@cKellyDesign what will we do about these?

Reply:
Not too sure unless we actually bring in `react-i18next`.